doc/network-zones.md

```diff
@@ -2,66 +2,179 @@
 ---
 
-# Network Zones configuration
-Network zones are used to hold DNS records for LXD networks.
+# How to configure network zones
 
-Each network can be related to up to 3 zones for:
+```{note}
+Network zones are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
- - Forward DNS records
- - IPv4 reverse DNS records
- - IPv6 reverse DNS records
+Network zones can be used to serve DNS records for LXD networks.
 
-This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4`
-and `dns.zone.reverse.ipv6` in network configuration. LXD will then be
-automatically managing forward and reverse records for all instances,
-network gateways and downstream network ports.
+You can use network zones to automatically maintain valid forward and reverse records for all your instances.
+This can be useful if you are operating a LXD cluster with multiple instances across many networks.
 
-To enable the built-in DNS server, `core.dns_address` must be set in the
-server configuration.
+Having DNS records for each instance makes it easier to access network services running on an instance.
+It is also important when hosting, for example, an outbound SMTP service.
+Without correct forward and reverse DNS entries for the instance, sent mail might be flagged as potential spam.
 
-The built-in DNS server only supports zone transfers through AXFR, it
-cannot be directly queried for DNS records. This means that this feature
-expects the use of an external DNS server (bind9, nsd, ...) which will
-transfer the entire zone from LXD, refresh it upon expiry and provide
-authoritative answers to DNS requests.
+Each network can be related to up to three zones for:
 
-Authentication for zone transfer is configured on a per-zone basis with
-peers defined in zone configuration and a combination of IP address
-matching and TSIG key based authentication.
+- Forward DNS records
+- IPv4 reverse DNS records
+- IPv6 reverse DNS records
 
-Zones belong to projects and are tied to the `networks` features of projects.
+LXD will then automatically manage forward and reverse records for all instances, network gateways and downstream network ports and serve those zones for zone transfer to the operator’s production DNS servers.
+
+## Generated records
+
+For example, if you configure a zone for forward DNS records for `lxd.example.net` for your network, it generates records that resolve the following DNS names:
+
+- For all instances in the network: `<instance_name>.lxd.example.net`
+- For the network gateway: `<network_name>.gw.lxd.example.net`
+- For downstream network ports (for network zones set on an uplink network with a downstream OVN network): `<profile_name>-<downstream_network_name>.uplink.lxd.example.net`
+
+You can check the records that are generated with your zone setup with the `dig` command.
+For example, running `dig @<DNS_server_IP> -p 1053 axfr lxd.example.net` might give the following output:
+
+```{bash}
```

Comment:
I manually changed the IP addresses here - do they look correct? Are the other numbers okay to include?